easybuild/tools/filetools.py

```diff
@@ -1054,12 +1054,17 @@ def convert_name(name, upper=False):
 
 
 def adjust_permissions(name, permissionBits, add=True, onlyfiles=False, onlydirs=False, recursive=True,
-                       group_id=None, relative=True, ignore_errors=False, skip_symlinks=True):
+                       group_id=None, relative=True, ignore_errors=False, skip_symlinks=None):
     """
     Add or remove (if add is False) permissionBits from all files (if onlydirs is False)
     and directories (if onlyfiles is False) in path
     """
 
+    if skip_symlinks is not None:
+        depr_msg = "Use of 'skip_symlinks' argument for 'adjust_permissions' is deprecated "
+        depr_msg += "(symlinks are always skipped now)"
```

Comment:
They aren't as much skipped as never followed, skipping them would involve not doing chgrp on the actual symlink, which is done through the lstat, lchown combo.

Reply:
OK, reworded in 9f5c584